### PR TITLE
Update setup

### PR DESCRIPTION
**Pull request by Kennytian**
Fix typo

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -5,7 +5,7 @@ if ! [ -x "$(command -v node)" ]; then
   echo 'Error: node is not installed.' >&2
 
   if ! [ -x "$(command -v brew)" ]; then
-    echo 'Go here to install: https://nodejs.org/en/download/'
```
Comment (Titozzz) [inline bin/setup]: This was actually intended to manually download node if brew is not here I think

**Comment by jamonholmgren** [comment]
Thanks @Kennytian! I don't necessarily want to require Homebrew. Rather, I'm giving someone the ability to manually download Node if they don't have it, as @Titozzz mentioned. Thoughts?

**Comment by Titozzz** [comment]
Seems ok to me

**Comment by jamonholmgren** [comment]
@Titozzz Do you mean you approve this PR?

**Comment by Titozzz** [comment]
No I think the author misunderstood and we can close it.

@Kennytian do you agree ?

**Comment by Kennytian** [comment]
@Titozzz  It's my fault,  I not understand `if ! [ -x "$(command -v brew)" ]; then` and `if [ -x "$(command -v brew)" ]; then`

line 7 is `if !`,  line 11 is `if`

All right, never mind this issue.  :-P

Thanks @Titozzz  and @jamonholmgren

**Comment by jamonholmgren** [comment]
Thanks anyway, @Kennytian; I hope you continue to contribute! <3

**Pull request closed without merging**